docs/source/configuration/reference.rst

```diff
@@ -187,6 +187,9 @@ Common configuration is a subdictionary that is a value of ``ext`` key in
     ``out`` and ``rewrite`` prompts (refer to IPython documentation for more 
     details) while ``in`` prompt is the default.
 
+    For wm it is a dictionary ``{output : theme_name}`` that maps the 
```

Comment:
Remembered that powerline has more then one binding that uses wm: also qtile and awesome. So “For wm (:ref:\`lemonbar <lemonbar-usage>\` only)”.